src/SmartComponents/AddSystemModal/AddSystemModal.js

```diff
@@ -92,7 +117,13 @@ class AddSystemModal extends Component {
                             eventKey={ 1 }
                             title="Baselines"
                         >
-                            <BaselinesTable hasSelect={ true }/>
+                            <BaselinesTable
+                                hasMultiSelect={ true }
+                                tableType='CHECKBOX'
+                                onSelect={ this.onSelect }
+                                baselineTableData={ baselineTableData }
+                                baselineListLoading={ baselineListLoading }
```

Comment:
I noticed that within BaselinesTable, `baselineListLoading` and `baselineDeleteLoading` are used in the same places and in the same way. This implies that they could be reduced down to a more generic `loading` variable and the interface here updated for that.